services/diagnostics/diagnostics_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-package app
+package diagnostics
```

Comment:
One thought I wanted to share: Outside of the code in e.g. the documentations this feature is called `telemetry`. There have been developers getting confused by this naming inconsistency.

Maybe this PR is a good opportunity to rename everything to `telemetry`?

Reply:
Very good point, I think you are totally right